.github-labelsrc.json

```diff
@@ -0,0 +1,170 @@
+[
```

Comment:
Nice. Would it make sense to have these published somewhere so we can just install them as a dependency and have them in sync more easily as a result?

Reply:
Hmm the problem with that is that the config needs the have the IDs of the labels for each repository.
I don’t know yet how we can solve that issue